lib/api/pokedex_api_spec/model/pokemon_details.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'pokemon_details.freezed.dart';
+
+part 'pokemon_details.g.dart';
+
+@freezed
+class PokemonDetails with _$PokemonDetails {
+  factory PokemonDetails({
+    @Default(0) @JsonKey(name: 'height') int height,
+    @Default(0) @JsonKey(name: 'weight') int weight,
+    @Default(0) @JsonKey(name: 'base_experience') int baseExp,
```

Comment:
- kindly complete the variable name
- kindly add a todo that this will be modified once other properties are added for about, stats and moves 🙂

Reply:
should I include all the property of about stats and moves for this file?

Reply:
yeah should be, but slowly add them once you're working already with specific tab pr 🙂

Reply:
should I make baseExp to baseExperience?